community-packages/package-list.json

```diff
@@ -1,5 +1,17 @@
 {
   "include": [
+    {
+      "repoSlug": "checkly/pulumi-checkly",
+      "schemaFile": "provider/cmd/pulumi-resource-checkly/schema.json"
+    },
+    {
+      "repoSlug": "EventStore/pulumi-eventstorecloud",
+      "schemaFile": "provider/cmd/pulumi-resource-eventstorecloud/schema.json"
+    },
+    {
+      "repoSlug": "grapl-security/pulumi-hcp",
+      "schemaFile": "provider/cmd/pulumi-resource-hcp/schema.json"
+    },
```

Comment:
Should we add buildkite here as well? https://github.com/pulumi/registry/pull/874

Reply:
We should! But I think after we merge that PR.